.github/workflows/publish-bicep.yaml

```diff
@@ -69,12 +76,14 @@ jobs:
           exit 1
 
       - id: get_radius_commit_hash
+        if: github.ref == 'refs/heads/main'
         name: Get commit hash (radius-project/radius)
         run: |
           echo "commit_hash_short=$(git rev-parse HEAD --short)" >> $GITHUB_OUTPUT
           echo "gh_uri=https://github.com/radius-project/radius/tree/$(git rev-parse HEAD)/hack/bicep-types-radius/generated" >> $GITHUB_OUTPUT
 
       - name: Clone https://github.com/radius-project/bicep
+        if: github.ref == 'refs/heads/main'
         uses: actions/checkout@v3
         with:
           token: ${{ secrets.GH_RAD_CI_BOT_PAT }}
```

Comment:
We discussed in meeting with @AaronCrawfis that token: ${{ secrets.GH_RAD_CI_BOT_PAT }} in the Clone https://github.com/radius-project/bicep step can/should be deleted.